README.md

```diff
@@ -5,13 +5,6 @@
 
 EasyPost is the simple shipping API. You can sign up for an account at <https://easypost.com>.
 
-Requirements
-------------
-
-* [Python](https://www.python.org) 2.7 or 3.3+ (or corresponding [PyPy](https://pypy.org) versions). Note that we only test on Python 2.7 and 3.5+; we strongly recommend against using 3.3.x or 3.4.x as they are no longer supported by many libraries.
```

Comment:
Can we keep the Requirements section? So new developers know which Python version their need to use our latest Python CL at least.

Reply:
I see your point, though I'm more of a fan of letting the code do the talking for items like this (requirements can change often), the metadata listed on pypi.org will list the minimum version as will pypi when you try to install it (it'll install a version that works for your env). This is especially important as the README is always the "current" documentation. For instance, people could still use our lib on Python2 but it'll be the last major version, but if we only say on the README it's usable via Python3.6 and newer, people may not try downloading if they're on Python2. With us bumping to 3.6, there's no explicit instructions needed anymore about versions or environments.

Reply:
This makes sense, but I think having some requirement detail would be helpful for future customers who want to use our product, so they can see the Python version requirement from the README file directly.

Reply:
Let's tackle this holistically if we are to document each libs version requirements. I have an internal ticket that we can track this work on.